sbioapputils/app_runner/dev_utils.py

```diff
@@ -31,26 +32,26 @@ def _run_pycodestyle(filename):
             
 
 def validate_yaml_stages(yaml_dict, style_check = False):
-    """Validating parameters from workflow yaml"""
-    stages = yaml_dict['stages']
-    
+    """Validating parameters from workflow yaml"""    
     valid_check = True
     invalid_stage = []
     invalid_path = []
     code_errors = []
     
-    for key in stages.keys():
-
+    for key in yaml_dict['stages'].keys():
```

Comment:
we shouldn't iterate this way, we should iterate like 
`for key, stage in yaml_dict['stages'].items():`

Reply:
I've changed this now